mod/tagger.php

```diff
@@ -148,24 +148,16 @@ function tagger_content(App $a) {
 
 	$post_id = Item::insert($arr);
 
-	if(! $item['visible']) {
-		$r = q("UPDATE `item` SET `visible` = 1 WHERE `id` = %d AND `uid` = %d",
-			intval($item['id']),
-			intval($owner_uid)
-		);
+	if (!$item['visible']) {
+		Item::update(['visible' => true], ['id' => $item['id']]);
```

Comment:
Do you not need 'uid' anymore for this update?

Reply:
The "id" field is unique. And there is no need for checking if this item really belongs to this user - this is done before.